scripts/hudson/narayana.sh

```diff
@@ -603,11 +603,13 @@ javax.sql.level = ALL
 org.apache.tomcat.tomcat-dbcp.level = ALL
 com.arjuna.level = ALL
 EOT
+    rm -rf narayana-tomcat
+    git clone https://github.com/jbosstm/narayana-tomcat.git
```

Comment:
I wonder that why we can not use https://github.com/web-servers/narayana-tomcat.git ?

Reply:
Just in case they are ever slot to merge one of our PR. I did have it set to that originally but then I found that test-utils/tomcat-jta order bug which they did merge quickly but I wanted to test this and so it failed. It's a bit like how we test against our 5_BRANCH of wildfly - does that make sense?

Reply:
OK, I understand.